### PR TITLE
Add minor changes

### PR DESCRIPTION
**Pull request by jtapia**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
 source 'https://rubygems.org'
 
 branch = ENV.fetch('SOLIDUS_BRANCH', 'master')
 gem 'solidus', github: 'solidusio/solidus', branch: branch
+gem 'solidus_auth_devise'
 
 if branch == 'master' || branch >= 'v2.0'
   gem 'rails-controller-testing', group: :test
 else
+  gem 'rails', '~> 4.2.0'
```
Comment (aitbw) [inline Gemfile]: :thinking: I don't know if we should include dependencies for Solidus versions lower than 2.0; after all, we stopped giving them support a long time ago, WDYT?

Comment (jtapia) [reply Gemfile]: got it, let me remove the support for old versions, thanks

Comment (jtapia) [reply Gemfile]: don here https://github.com/solidusio-contrib/solidus_abandoned_carts/pull/1/commits/93983791ee408fa3a2e3691e3b34181ad4ddc8db

Comment (aitbw) [reply Gemfile]: You can also safely remove the `if branch == 'master' || branch >= 'v2.0'` conditional, as we're only testing Solidus v2.0+

Comment (jtapia) [reply Gemfile]: right, done here https://github.com/solidusio-contrib/solidus_abandoned_carts/pull/1/commits/e2c81a6c47e574a9abf8eca23c2525bb04dc165c

File: .travis.yml [hunk 2]
```diff
@@ -5,29 +5,23 @@ rvm:
   - 2.3.1
 env:
   matrix:
-    - SOLIDUS_BRANCH=v1.0 DB=postgres
-    - SOLIDUS_BRANCH=v1.1 DB=postgres
-    - SOLIDUS_BRANCH=v1.2 DB=postgres
-    - SOLIDUS_BRANCH=v1.3 DB=postgres
-    - SOLIDUS_BRANCH=v1.4 DB=postgres
     - SOLIDUS_BRANCH=v2.0 DB=postgres
```
Comment (aitbw) [inline .travis.yml]: You can remove support for v2.0 to v2.3 for both `postgres` and `mysql`; that should speed up the CI considerably.

File: lib/solidus_abandoned_carts/factories.rb [hunk 3]
```diff
@@ -1,6 +1,8 @@
-FactoryGirl.define do
+# frozen_string_literal: true
+
+FactoryBot.define do
   # Define your Spree extensions Factories within this file to enable applications, and other extensions to use and override them.
   #
   # Example adding this to your spec_helper will load these Factories for use:
-  # require 'solidus_abandoned_carts/factories'
```
Comment (aitbw) [inline lib/solidus_abandoned_carts/factories.rb]: Is this change necessary in here? :thinking:

File: .travis.yml (hunk 2 above)
Comment (jtapia) [reply .travis.yml]: done, thanks https://github.com/solidusio-contrib/solidus_abandoned_carts/pull/1/commits/18496e9e23d172cd444e43eb994139d1da759495#diff-354f30a63fb0907d4ad57269548329e3L8

File: lib/solidus_abandoned_carts/factories.rb (hunk 3 above)
Comment (jtapia) [reply lib/solidus_abandoned_carts/factories.rb]: my bad, fixed

File: app/.rubocop.yml [hunk 4]
```diff
@@ -0,0 +1,249 @@
+# Relaxed.Ruby.Style
```
Comment (aitbw) [inline app/.rubocop.yml]: This file should be in the root folder, not inside `app/`; I also think this file should be included in a different, isolated commit, or maybe in a different PR, what do you think?

Comment (jtapia) [reply app/.rubocop.yml]: yes, it's now on project root here https://github.com/solidusio-contrib/solidus_abandoned_carts/pull/1/files#diff-11a0d7906801d9dea0eccb85667ad811R1

**Comment by jtapia** [comment]
@aitbw @kennyadsl I think there's an issue with travis, maybe related with the ownership change
